src/parse-env.js

```diff
@@ -0,0 +1,32 @@
+module.exports = env => {
+  const {
+    AIRTABLE_BASE_ID,
+    AIRTABLE_API_KEY,
+    ALLOWED_METHODS,
+    PORT = 3000,
+    READ_ONLY,
+  } = env;
+
+  if (
+    typeof AIRTABLE_BASE_ID === 'undefined' ||
+    typeof AIRTABLE_API_KEY === 'undefined'
+  ) {
+    throw new TypeError(
+      'Please provide AIRTABLE_BASE_ID and AIRTABLE_API_KEY as environment variables.'
+    );
+  }
+
+  const config = {
+    airtableApiKey: AIRTABLE_API_KEY,
+    airtableBaseId: AIRTABLE_BASE_ID,
+    port: PORT,
+  };
+
+  if (READ_ONLY === 'true') {
```

Comment:
Since we now have defaults, we can leave out the `allowedMethods` unless something is explicitly passed in.